<launch>
    <node name="device_reconfigure" pkg="akara_controller" type="device_configure" output="screen">
        <rosparam command="load" file="$(find akara_controller)/config/tnpa.yaml" />
    </node>
</launch>
